petab/sbml.py

```diff
@@ -106,9 +111,12 @@ def constant_species_to_parameters(sbml_model: libsbml.Model) -> list:
     # Remove from reactants and products
     for reaction in sbml_model.getListOfReactions():
         for speciesId in transformables:
-            reaction.removeReactant(speciesId)
-            reaction.removeProduct(speciesId)
-            reaction.removeModifier(speciesId)
+            while reaction.removeReactant(speciesId):
```

Comment:
add comment that the loop is because the functions always only remove 1? helps code maintenance later.

Reply:
done